packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md

```diff
@@ -0,0 +1,345 @@
+
+A trusted server handles authenticated requests after the application obtains an access token. The incoming subject token (access token) is used to evaluate a subject.
+
+The following example walks you through how to set up a trusted server, scopes, access policy, and rule. Then, you can make the token exchange request with the trusted server.
+
+### Add a trusted server
+
+Add the custom authorization server that you used in the previous flow as a trusted server of the authorization server that handles the token exchange in the trusted server flow.
+
+1. In the Admin Console, go to **Security** > **API**.
+1. Select the edit (pencil) icon on the right of the custom authorization server that you want to associate a trusted server with for this example.
+1. In the **Trusted servers** section, click **Add Server**.
+1. In the **Search** field, enter the name of the authorization server that you used in the token exchange flow previously. Matching results appear in a list. If more than 20 results appear, you can click **Show more**.
+1. Click **Add** beside the authorization server, and then click *Done**. The authorization server appears in the **Trusted servers** section.
+
+### Create an access policy and rule
+
+Create an access policy and rule so that the service app can access API2 using a different authorization server than the one that granted the original access token for the user.
+
+1. Select the Access Policies tab, and then click Add New Access Policy to add a policy that allows the service app to access API2.
+1. In the Add Policy dialog that appears, enter the following:
+
+   * **Name:** Enter a name for the new access policy (**Access API2**).
+   * **Description:** Enter a description.
+   * **Assign to:** Select **The following clients**, start typing **API1**, and then select it from the list that appears.
+
+1. Click **Create Policy**.
+1. Click **Add Rule** and in the dialog that appears, enter the following:
+
+   * Name: Enter a name for the rule (**API1 to API2**).
+   * AND Scopes requested: Select The following scopes, start typing **api:access:read** and select it from the list that appears. Repeat for **api:access:write** and select it from the list.
+
+1. Click **Create rule**.
+
+### Flow specifics with a trusted server
+
+Perform the requests in the previous [Flow specifics](#flow-specifics) section. When you reach the [token exchange request from the service app](#token-exchange-request-from-service-app-to-api) request, it should look like the following:
+
+> **Note:** Use the Base64 encoded client ID and secret for the service app that you generated during the service app creation earlier. Place that in the Authorization header. See the following example for the format.
+
+```curl
+curl --location --request POST \
+  --url 'https://${yourOktaDomain}/oauth2/{trustedauthServerId}/v1/token' \
```

Comment:
Maybe better to change to: 
-> `--url 'https://${yourOktaDomain}/oauth2/{trustedAuthServerId}/v1/token' \`

Uppercase for A in trusted**A**uthServerId.